### DISCUSSION TITLE
steam: crashes with private-tmp (dbus)

### DISCUSSION THREAD

**Discussion by amano-kenji** [netblue30/firejail#5743] [Q&A] [answered]
> /home/user/.local/share/Steam/steam.sh: line 798:   362 Segmentation fault      "$STEAMROOT/$STEAMEXEPATH" "$@"

Without firejail, I can execute /usr/bin/steam.

**Comment by kmk3** [comment, marked as answer]
Basic debugging information is missing; please follow the bug report template
in a new issue:

* <https://github.com/netblue30/firejail/issues/new?template=bug_report.md>

Edit: Moved to:

* <https://github.com/netblue30/firejail/issues/5745>
